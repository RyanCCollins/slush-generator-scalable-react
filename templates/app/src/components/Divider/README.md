## Divider Component
A component that acts as a section divider.

### Example

```js
<Divider />
```
