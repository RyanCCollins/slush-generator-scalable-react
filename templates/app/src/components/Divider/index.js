import React from 'react';
import cssModules from 'react-css-modules';
import styles from './index.module.scss';

const Divider = () => (
  <span className={styles.divider} />
);


export default cssModules(Divider, styles);
