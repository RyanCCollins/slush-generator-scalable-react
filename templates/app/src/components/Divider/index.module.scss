.divider {
  background-color: #7d97ad;
  width: 150px;
  height: 3px;
  display: block;
  margin: 35px 0;
  margin-right: auto;
  margin-left: auto;
}
