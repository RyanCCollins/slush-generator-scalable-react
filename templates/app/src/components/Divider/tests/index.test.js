import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import React from 'react';
import Divider from '../index';

describe('<Divider />', () => {
  it('should render with default props', () => {
    const wrapper = shallow(
      <Divider />,
    );
    expect(shallowToJson(wrapper)).toMatchSnapshot(); // eslint-disable-line
  });
});
