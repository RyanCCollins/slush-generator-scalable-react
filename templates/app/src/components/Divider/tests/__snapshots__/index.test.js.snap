exports[`<Divider /> should render with default props 1`] = `<span />`;
